src/scrabble.js

```diff
@@ -1,6 +1,133 @@
-class Scrabble {
-  // Write your implementation here
+let doubleWord = false
+let doubleLetterRef = []
+let tripleWord = false
+let tripleLetterRef = []
+let points = 0
+const commonLetters = [['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'], ['D', 'G'], ['B', 'C', 'M', 'P'], ['F', 'H', 'V', 'W', 'Y'], ['K']]
+const rareLetters = [['J', 'X'], ['Q', 'Z']]
+const double = /\{[A-Z{}[\]]*\}/g
+const triple = /\[[A-Z{}[\]]*\]/g
+
+// functions called directly by Scrabble class
+function noInput (type) {
+  return type === null
```

Comment:
So this is a little bit of a quirk with javascript (and using === which forces types to be the same). `null` is a value that developers set. However, if nothing is set for the word value it wouldn't be `null` instead it would be `undefined`.

```js
let a = null
let b = undefined

a === null // true
b === null //false
b == null // true
```